scripts/integration-test.sh

```diff
@@ -1,6 +1,16 @@
 #!/bin/bash
 set -eo pipefail
 
+XVFB_RUN() {
+    if hash xvfb-run 2>/dev/null
+    then
+        xvfb-run "$@"
+    else
+        echo "WARN: xvfb-run must be installed to run Chrome headless"
```

Comment:
nit: update to "run Chrome with extensions enabled on a headless server" or something to disambiguate headless driver from headless operation